### PR TITLE
RELATED: PSDK-165 automate release process

### PR DESCRIPTION
**Pull request by hkad98**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: scripts/bump_version.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import tomllib
+from pathlib import Path
+
+_ROOT_DIR = Path(__file__).resolve().parent.parent
+
+
+def get_version(content: dict) -> list[int]:
+    current_version = content.get("tool", {}).get("tbump", {}).get("version", {}).get("current")
+    if current_version is None:
+        raise ValueError("No current version found while reading ")
```
Comment (tomaso) [inline scripts/bump_version.py:12]: This looks like unfinished sentence.

File: .github/workflows/bump-version.yaml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+          uses: actions/checkout@v4
+        - uses: actions/setup-python@v4
+          with:
+            python-version-file: '.python-version'
+            cache: 'pip'
+        - name: Bump version
+          id: bump
+          run: |
+            NEW_VERSION=$(python ./scripts/bump_version.py ${{ github.event.inputs.bump_type }})
+            echo "::set-output name=new_version::$NEW_VERSION"
```
Comment (tomaso) [inline .github/workflows/bump-version.yaml:36]: This syntax is deprecated and will issue warnings in your output.
It should look like this:
echo "new_version=$NEW_VERSION" >>$GITHUB_OUTPUT

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by tomaso — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR425

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul/237/ : FAILURE in 1m 06s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/387/ : SUCCESS in 55s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/391/ : SUCCESS in 9s

**Pull request merged**